### PR TITLE
removed amp_store default when creating a new project

### PR DESCRIPTION
**Pull request by pwills11**
*Issue number of the reported bug or feature request: #<number>*

**Describe your changes**
A clear and concise description of the changes you have made.

**Testing performed**
Describe the testing you have performed to ensure that the bug has been addressed, or that the new feature works as planned.

**Additional context**
Add any other context about your contribution here.

### CODE REVIEW

**Comment by pwills11** [comment]
This change removed the default selection of amp_store when a project is L2 or greater because a bug was created when a user would 
1. choose to sync their project with prod when the project doesn't exist in prod
2. when the project is created in prod in order to sync, the autoselection of amp_store was set to true
3. when the user's dev project does not have the amp_store selected, then there will not be a amp_pvf value to sync with the prod project

the result: this leaves the prod project with amp_pvf empty and amp_store opt in set to true, resulting in the following bad request error:
https://github.com/bloomberg/pybossa/blob/c0c58d445790a64b534320289b97b69553721a2b/pybossa/repositories/project_repository.py#L203

**Review by XiChenn — APPROVED** [review]
lgtm

File: pybossa/data_access.py [hunk 1]
```diff
@@ -198,6 +198,5 @@ def set_default_amp_store(project):
     annotation_config = project['info'].get('annotation_config', {})
     input_data = project['info'].get('data_classification', {}).get('input_data') or ''
     output_data = project['info'].get('data_classification', {}).get('output_data') or ''
-    # Set amp_store default to False if 'L1' is in input or output data classifications.
```
Comment (ywrsusan) [inline pybossa/data_access.py:201]: Why do we remove the default amp_store? I thought we only want to remove default settings when syncing, but when user create project through API or other method, we need the default amp_store (opt-out for L1, and opt-in for L2-L4). Am I missing anything?

Comment (XiChenn) [reply pybossa/data_access.py:201]: I guess the main issue before this change was that when a L2-L4 project with no "amp_pvf" configured, synchronizing project to Prod with amp_store True results an error, causing the sync failed. See Paxton's comment in the PR for detail.

**Pull request merged**